CHANGELOG.rst

```diff
@@ -5,6 +5,27 @@ amazon.aws Release Notes
 .. contents:: Topics
 
 
+v5.6.0
```

Comment:
I see only bug fixes in this release. So I think that the version should be 5.5.1.